CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [unreleased]
 
+### Changed
+
+- AI data is synchronizable between devices via the new functions `syncdata_bytes` and `synchronize`.
+
 ## 2.0.0 - 2021/06/28
```

Comment:
```suggestion
## 2.0.0 - 2021/08/24
```

Reply:
No this is like this in the release branch and repository so I don't want to change it.
Next time we can do a pre release and put the date only when we push to the release branch.

Reply:
What really counts is the tag